urwid/__init__.py

```diff
@@ -194,10 +194,16 @@
     scale_bar_values,
 )
 
-from . import event_loop, widget
+from . import display, event_loop, widget
 
 from urwid.treetools import ParentNode, TreeListBox, TreeNode, TreeWalker, TreeWidget, TreeWidgetError  # isort: skip
 
+# Optional displays with external dependencies
+try:
+    from .display import curses as curses_display  # backward compatibility
```

Comment:
this way we have full backward compatibility without extra files.
Even `PendingDeprecation` is too much for such common module at this moment